README.md

```diff
@@ -66,16 +78,16 @@ Returns the list of tuples below.
  ('EOS_TOKEN', 0.0)]
 ```
 
-Positive attribution numbers indicate a word contributes positively towards the predicted class, negative numbers indicate a word contributes negatively towards the predicted class. Here we can see that **I love you** gets the most attention.
+Positive attribution numbers indicate a word contributes positively towards the predicted class, while negative numbers indicate a word contributes negatively towards the predicted class. Here we can see that **I love you** gets the most attention.
 
-In case you want to know what the predicted class actually is:
+You can use `predicted_class_index` in case you'd want to know what the predicted class actually is:
 
 ```python
 >>> cls_explainer.predicted_class_index
-array(1)
+array(1) # this is a pretty ambiguous output, will this make sense to people?
```

Comment:
Added as a comment so you can address it

Reply:
I'll fix this in a future update